rfcs/convergence/authoring-stories.md

```diff
@@ -4,6 +4,10 @@
 
 _List contributors to the proposal:_ @hotell, @miroslavstastny, @ling1726, @andrefcdias, @PeterDraex
 
+<!--
+command to generate TOC:
+npx markdown-toc -i rfcs/convergence/authoring-stories.md
```

Comment:
It might be better to just use+suggest `doctoc` here since it's already used for other TOCs in our repo.
```suggestion
yarn doctoc --github rfcs/convergence/authoring-stories.md
```